pdudaemon/listener.py

```diff
@@ -48,7 +48,7 @@ def parse_tcp(data):
 
 def parse_http(data, path):
     args = Args()
-    delay = None
+    _delay = None
```

Comment:
This line should just be removed.

Reply:
noted

Reply:
removed, together with other unused variable (Refer to commit with 'F481: unused variable' commit message)